test/utils/pickers/describeJalaliAdapter/testCalculations.ts

```diff
@@ -19,6 +19,16 @@ export const testCalculations: DescribeJalaliAdapterTestSuite = ({ adapter }) =>
 
     expect(adapter.isEqual(testDateIso, anotherDate)).to.equal(true);
     expect(adapter.isEqual(null, null)).to.equal(true);
+    expect(adapter.isEqual(testDateIso, null)).to.equal(false);
+  });
+
+  it('Method: isValid', () => {
+    expect(adapter.isValid(testDateIso)).to.equal(true);
+    if (adapter.lib !== 'moment-jalaali') {
+      // moment-jalaali just throw errors
+      expect(adapter.isValid(null)).to.equal(false);
+      expect(adapter.isValid(adapter.date('invalid'))).to.equal(false);
+    }
```

Comment:
Not sure it works, or it's usefull
```suggestion
    if (adapter.lib !== 'moment-jalaali') {
      expect(adapter.isValid(null)).to.equal(false);
      expect(adapter.isValid(adapter.date('invalid'))).to.equal(false);
    } else {
      expect(() => adapter.isValid(null)).to.throw();
      expect(() => adapter.isValid(adapter.date('invalid'))).to.throw();
    }
```

Reply:
Pushed the following change: https://github.com/mui/mui-x/pull/13969/commits/79e1bcf6f7b45f281d0d5b7a759a154db42b83c8
Your suggestion didn't work. 🙈